jetstream/bookmarks-toolbar-default-on.toml

```diff
@@ -5,12 +5,12 @@ overall = ['bookmarks_bar_pinned_on', 'bookmarks_bar_pinned_off', 'bookmarks_bar
 
 
 [metrics.bookmarks_bar_pinned_on]
-select_expression = "CAST(COALESCE(SUM(CASE WHEN(SELECT CAST(COUNT(t) AS BOOL) FROM UNNEST(payload.processes.parent.keyed_scalars.browser_ui_toolbar_widgets) t WHERE t.key LIKE 'bookmarks-bar_pinned_on' AND t.value) THEN 1 ELSE 0 END),0) > 0 AS INT)"
+select_expression = '''CAST(COALESCE(SUM(CASE WHEN(SELECT CAST(COUNT(t) AS BOOL) FROM UNNEST(payload.processes.parent.keyed_scalars.browser_ui_toolbar_widgets) t WHERE t.key LIKE r'bookmarks-bar\_pinned\_on' AND t.value) THEN 1 ELSE 0 END),0) > 0 AS INT)'''
```

Comment:
This `LIKE` pattern doesn't have any `%` wildcards, so I suppose the underscores could have been meant as wildcards, but I think it's unlikely given the context.